<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MineData</class>
 <widget class="QMainWindow" name="MineData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>954</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(16, 5, 44);
background-color: rgb(51, 23, 84);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="pandasTv">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>561</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="upload_button">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>650</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:rgb(255, 147, 203);
	border-radius:10px;
	border-style:outset;
}
QPushButton:hover {
            background-color: rgb(246, 252, 126);
        }
QPushButton:focus {
            background-color: rgb(90, 71, 173);
        }
</string>
    </property>
    <property name="text">
     <string>upload</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_button">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>650</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:rgb(255, 147, 203);
	border-radius:10px;
	border-style:outset;
}
QPushButton:hover {
            background-color: rgb(246, 252, 126);
        }
QPushButton:focus {
            background-color: rgb(90, 71, 173);
        }
</string>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QLabel" name="dataset_info">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width:2px;
border-radius:10px;
border-width:2px;
border-style:outset;
border-color: rgb(148, 88, 122);
font: 350 10pt &quot;Bahnschrift SemiLight&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="info_label">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 13pt &quot;Bahnschrift SemiBold&quot;;
color: rgb(255, 105, 185);</string>
    </property>
    <property name="text">
     <string>Information about the dataset</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>100</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 13pt &quot;Bahnschrift SemiBold&quot;;
color: rgb(255, 105, 185);</string>
    </property>
    <property name="text">
     <string>Information about the column</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="column_info">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>120</y>
      <width>361</width>
      <height>251</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width:2px;
border-radius:10px;
border-width:2px;
border-style:outset;
border-color: rgb(148, 88, 122);
background-color: rgb(255, 255, 255);
font: 350 10pt &quot;Bahnschrift SemiLight&quot;;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="centerOnScroll">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>931</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget{
color:white;
border-radius:10px;
border-color:rgb(0,0,0);
border-style:outset;
}
QTabWidget::pane { /* The tab widget frame */
    border-top: 2px solid rgb(16, 5, 44);
	border-left: 2px solid rgb(16, 5, 44);
	border-right: 2px solid rgb(16, 5, 44);
	border-bottom: 2px solid rgb(16, 5, 44);	
	border-width:2px;
	border-style:outset;
	border-radius:12px;
}

QTabWidget::tab-bar {
    left: 5px; /* move to the right by 5px */
}

/* Style the tab using the tab sub-control. Note that
    it reads QTabBar _not_ QTabWidget */
QTabBar::tab {
    background:  rgb(63, 50, 105);
    border: 2px solid  rgb(16, 5, 44);
    border-bottom-color:  rgb(16, 5, 44); /* same as the pane color */
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
    min-width: 8ex;
    padding: 2px;
	color:white;
	width:50px;
}

QTabBar::tab:selected, QTabBar::tab:hover {
    background: rgb(99,78,163);
color:white
}

QTabBar::tab:selected {
    border-color:rgb(16, 5, 44) ;
color:white;
    border-bottom-color: rgb(99,78,163); /* same as pane color */
}

QTabBar::tab:!selected {
    margin-top: 2px; /* make non-selected tabs look smaller */
}

/* make use of negative margins for overlapping tabs */
QTabBar::tab:selected {
    /* expand/overlap to the left and right by 4px */
    margin-left: -4px;
    margin-right: -4px;
}

QTabBar::tab:first:selected {
    margin-left: 0; /* the first selected tab has nothing to overlap with on the left */
}

QTabBar::tab:last:selected {
    margin-right: 0; /* the last selected tab has nothing to overlap with on the right */
}

QTabBar::tab:only-one {
    margin: 0; /* if there is only one tab, we don't want overlapping margins */
}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="partie1">
     <attribute name="title">
      <string>Partie 1</string>
     </attribute>
     <widget class="QRadioButton" name="boxplot_radio">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>10</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Boxplot</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>-20</y>
        <width>351</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="histogramme_radio">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>40</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>histogramme</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="scatter_radio">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>70</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Scatter Plot</string>
      </property>
     </widget>
     <widget class="QComboBox" name="scatter_column1">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>130</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 147, 203);
	border-radius:10px;
	border-style:outset;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="scatter_column2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>100</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 147, 203);
	border-radius:10px;
	border-style:outset;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="draw">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>190</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:rgb(255, 147, 203);
	border-radius:10px;
	border-style:outset;
}
QPushButton:hover {
            background-color: rgb(246, 252, 126);
        }
QPushButton:focus {
            background-color: rgb(90, 71, 173);
        }
</string>
      </property>
      <property name="text">
       <string>Draw graph</string>
      </property>
     </widget>
     <widget class="QLabel" name="logo_label">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>30</y>
        <width>221</width>
        <height>181</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>bigger_logo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="correlation_label">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>160</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="partie2">
     <attribute name="title">
      <string>Partie 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>954</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
